.github/workflows/fe_build.yml

```diff
@@ -0,0 +1,81 @@
+name: FE Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      publisher_payload:
+        description: Publisher payload
+        required: true
+      env:
+        description: Environment
+        required: true
+
+run-name: 'FE Build [env: ${{ inputs.env }}]'
+
+concurrency:
+  group: fe_build_${{ inputs.env }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Setup
+        uses: ./.github/actions/setup
+
+      - name: TurboRepo local server
+        uses: felixmosh/turborepo-gh-artifacts@v2
+        with:
+          server-token: 'local'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Prepare
+        run: SKIP_DRUPAL_INSTALL=1 pnpm turbo:prep
+        env:
+          TURBO_API: 'http://127.0.0.1:9080'
+          TURBO_TOKEN: 'local'
+          TURBO_TEAM: 'local'
+
+      - name: Publisher
+        uses: AmazeeLabs/publisher-action@4371b4a90a9fe2c86248e76c0f428e39abf6b610
```

Comment:
could be automated as tagged version

Reply:
Yes, I even tried to use tags first. But it's so annoying to update them :)